src/crypto/tls/handshake_client.go

```diff
@@ -73,6 +73,8 @@ func (c *Conn) makeClientHello() (*clientHelloMsg, ecdheParameters, error) {
 		supportedCurves:              config.curvePreferences(),
 		supportedPoints:              []uint8{pointFormatUncompressed},
 		secureRenegotiationSupported: true,
+		// for TLS 1.2, delegatedCredentials are not supported
+		delegatedCredentialSupported: false,
```

Comment:
`false` is the default, it does not have to be set.

Reply:
Removing it ;)